integrationtests/controller/bundle/label_migration_test.go

```diff
@@ -0,0 +1,96 @@
+package bundle
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/rancher/fleet/integrationtests/utils"
+	"github.com/rancher/fleet/pkg/apis/fleet.cattle.io/v1alpha1"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var _ = Describe("Bundle label migration", func() {
+	BeforeEach(func() {
+		var err error
+		namespace, err = utils.NewNamespaceName()
+		Expect(err).ToNot(HaveOccurred())
+		Expect(k8sClient.Create(ctx, &corev1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{Name: namespace},
+		})).ToNot(HaveOccurred())
+
+		_, err = utils.CreateCluster(ctx, k8sClient, "cluster", namespace, nil, namespace)
+		Expect(err).NotTo(HaveOccurred())
+
+		DeferCleanup(func() {
+			Expect(k8sClient.Delete(ctx, &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: namespace}})).ToNot(HaveOccurred())
+		})
+	})
+
+	createBundle := func(name string, labels map[string]string) {
+		bundle := &v1alpha1.Bundle{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      name,
+				Namespace: namespace,
+				Labels:    labels,
+			},
+			Spec: v1alpha1.BundleSpec{
+				BundleDeploymentOptions: v1alpha1.BundleDeploymentOptions{
+					DefaultNamespace: "default",
+				},
+				Targets: []v1alpha1.BundleTarget{
+					{
+						BundleDeploymentOptions: v1alpha1.BundleDeploymentOptions{
+							TargetNamespace: "targetNs",
+						},
+						Name:        "cluster",
+						ClusterName: "cluster",
+					},
+				},
+			},
+		}
+		Expect(k8sClient.Create(ctx, bundle)).ToNot(HaveOccurred())
+	}
+
+	DescribeTable("should remove deprecated label after migration",
+		func(bundleName string, initialLabels map[string]string, shouldHaveDisplayLabel bool) {
```

Comment:
Where is argument `shouldHaveDisplayLabel` used? :thinking: 
(same question in Gitjob tests)